build.gradle

```diff
@@ -30,6 +30,9 @@ if (System.env.CIRCLE_TEST_REPORTS) {
 group 'com.palantir.gradle.docker'
 version System.env.CIRCLE_TAG ?: gitVersion()
 
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
```

Comment:
i believe that targetC inherits from sourceC? if that is true, the first line should suffice.